pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -121,6 +132,25 @@ main() {
 	Match *
 	Port 29230"
 
+    # values are references to variables, they should not be dereferenced here
+    # shellcheck disable=SC2034
+    local -rA aro_configs=(
+        ["rp_config"]="aro_rp_conf_file"
+        ["fluentbit"]="fluentbit_conf_file"
+        ["mdsd"]="mdsd_config_version"
+        ["static_ip_address"]="static_ip_addresses"
+    )
+
+    # shellcheck disable=SC2034
+    # use default podman network with range 10.88.0.0/16
+    local -rA static_ip_addresses=(
+        ["rp"]="10.88.0.2"
```

Comment:
Could you point me why we need static ip? Also I see the gateway and RP is using the same IP, is this expected?

Reply:
Static IP addresses are being assigned to containers to use that in the Geneva Monitor to identify the MISE endpoint in a generic way across all VMs in VMSS. 
As we are only monitoring MISE endpoints in this way and  Gateway & RP services are deployed in different VMSS, we have kept their IPs as same.

Reply:
Thanks, could you also add comment in the code, so others can understand why's this?